src/librbd/managed_lock/GetLockerRequest.h

```diff
@@ -9,6 +9,10 @@
 
 class Context;
 
+namespace librados {
+  class IoCtx;
```

Comment:
Nit: no indentation within namespaces. You could collapse to a single line, though.